src/OrchardCore.Modules/OrchardCore.Navigation/Startup.cs

```diff
@@ -6,13 +7,21 @@ namespace OrchardCore.Navigation
 {
     public class Startup : StartupBase
     {
+        private readonly IConfiguration _configuration;
+
+        public Startup(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
         public override void ConfigureServices(IServiceCollection services)
         {
             services.AddNavigation();
 
             services.AddScoped<IShapeTableProvider, NavigationShapes>();
             services.AddScoped<IShapeTableProvider, PagerShapesTableProvider>();
             services.AddShapeAttributes<PagerShapes>();
+            services.Configure<PagerOptions>(options => _configuration.GetSection("OrchardCore:OrchardCore_Pager").Bind(options));
```

Comment:
Hmm, maybe `OrchardCore:OrchardCore_Navigation` to follow the section convention = module/feature name.

Then maybe the following if we need other options still related to the Navigation module.

    "OrchardCore_Navigation": {
        "PagerOptions": {
            "PageSize": 10,
            "MaxPageSize": 100,
            "MaxPagedCount": 100
        }
    },

If so, as I remember we may not need an additional `.GetSection("PagerOptions")` to the above `Bind()` as it will see the matching `PagerOptions` property, but needs to be tried.

Reply:
@jtkech I made the requested changed

FYI the both of the following worked
```
_configuration.GetSection("OrchardCore:OrchardCore_Navigation").GetSection("PagerOptions").Bind(opt);
```

and 
```
_configuration.GetSection("OrchardCore:OrchardCore_Navigation:PagerOptions").Bind(opt);
```

but this did NOT work
```
_configuration.GetSection("OrchardCore:OrchardCore_Navigation:PagerOptions").Bind(opt);
```

Reply:
Okay cool

Don't see any difference in your 2 last examples